### PR TITLE
chore: shorten `flow-huge.spec.js` test case using `.repeat`

### PR DESCRIPTION
**Pull request by Yokozuna59**
## :bookmark_tabs: Summary

Just use `''.repeat(20000);` instead of hardcoding nodes.

When the file is opened in editors, it stops highlighting and becomes laggy.

Make sure you

- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

File: packages/mermaid/src/diagrams/flowchart/parser/flow-huge.spec.js
Comment (aloisklink) [inline packages/mermaid/src/diagrams/flowchart/parser/flow-huge.spec.js:17]:
```suggestion
      const nodes = ('A-->B;B-->A;'.repeat(415) + 'A-->B;').repeat(57) + 'A-->B;B-->A;'.repeat(275);
```

If we want to make the string the exact same as before, we have to use this weird pattern above! (Sometimes the string has an `A-->B;A-->B;` next to each other).

**Review by aloisklink — CHANGES_REQUESTED** [review]
I believe this test fails after your changes, because the string is now shorter:

```
 FAIL  packages/mermaid/src/diagrams/flowchart/parser/flow-huge.spec.js > [Text] when parsing > it should handle huge files > it should handlehuge diagrams
AssertionError: expected 40000 to be 47917 // Object.is equality

- Expected
+ Received

- 47917
+ 40000

 ❯ packages/mermaid/src/diagrams/flowchart/parser/flow-huge.spec.js:24:28
     22| 
     23|       expect(edges[0].type).toBe('arrow_point');
     24|       expect(edges.length).toBe(47917);
       |                            ^
     25|       expect(Object.keys(vert).length).toBe(2);
     26|     });
```

See my comments on how to fix this:

File: packages/mermaid/src/diagrams/flowchart/parser/flow-huge.spec.js
Comment (Yokozuna59) [reply packages/mermaid/src/diagrams/flowchart/parser/flow-huge.spec.js:18]: I tried taking a deep look on actual code, but highlighting wasn't working, so I opened vim and tried to go over the code but it was in one line :)

**Review by aloisklink — APPROVED** [review]
Awesome! The test works now :)

The next thing would be updating https://github.com/mermaid-js/mermaid/blob/735aceb37a3a5907c97e9de0ca243fa6ae1b31a4/cypress/platform/huge.html to do the same thing.

@Yokozuna59, since this PR is very small and low-risk, I'm happy for you to merge this PR now (I'll let you decide when to merge it since you've assigned yourself to this PR).

**Comment by Yokozuna59** [comment]
> The next thing would be updating https://github.com/mermaid-js/mermaid/blob/735aceb37a3a5907c97e9de0ca243fa6ae1b31a4/cypress/platform/huge.html to do the same thing.

Sure, but the default `maxTextSize` is 50000, should I also update it? The current test size is 287502 :)
I don't think it would be able to render it with that size. Even the browser says so :) :

image
 
> @Yokozuna59, since this PR is very small and low-risk, I'm happy for you to merge this PR now (I'll let you decide when to merge it since you've assigned yourself to this PR).

No worries.

**Comment by aloisklink** [comment]
> > The next thing would be updating https://github.com/mermaid-js/mermaid/blob/735aceb37a3a5907c97e9de0ca243fa6ae1b31a4/cypress/platform/huge.html to do the same thing.
> 
> Sure, but the default `maxTextSize` is 50000, should I also update it? The current test size is 287502 :) I don't think it would be able to render it with that size. Even the browser says so :) :

It looks like rendering a "Maximum text size in diagram exceeded" error message is the expected behavior, see https://github.com/mermaid-js/mermaid/pull/1221, so we shouldn't change `maxTextSize`.

**Comment by Yokozuna59** [comment]
@aloisklink I see, then is the PR ready to be merged?

**Review by aloisklink — APPROVED** [review]
> @aloisklink I see, then is the PR ready to be merged?

Yep, everything looks great to me :) (it's even better now with the `cypress/platform/huge.html` changes).

I'll merge it myself, since this PR has been open for a few days already, but since it's only test changes, we don't really need reviews from anybody else.

**Pull request merged**
